tethne/tests/test_writers_corpus.py

```diff
@@ -31,6 +33,26 @@ def test_write_documents(self):
         self.assertGreater(os.path.getsize(self.dpath), 0)
         self.assertGreater(os.path.getsize(self.mpath), 0)
 
```

Comment:
Be sure to leave two blank lines above any root-level function or class declaration.